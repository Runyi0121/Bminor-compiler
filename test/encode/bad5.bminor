'?silly characters does not work?'