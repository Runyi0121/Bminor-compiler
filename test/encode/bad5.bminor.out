does not have quotation in the front.
There is an error in the original string.
